benchmarking/commons/utils.py

```diff
@@ -73,7 +73,7 @@ def find_or_create_requirements_txt(
 ) -> Path:
     if requirements_fname is None:
         requirements_fname = "requirements.txt"
-    files = list(entry_point.parent.rglob("requirements*.txt"))
+    files = list(entry_point.parent.glob("requirements*.txt"))
```

Comment:
[Sagemaker frameworks](https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/using_pytorch.html) pull requirements directly from the directory of the entrypoint therefore we should only look for requirements in the entrypoint dir rather than the whole tree.

Reply:
You are right, I think I misunderstood the docs.